app/main.py

```diff
@@ -1 +1,61 @@
-# write your code here
+class SoftwareEngineer:
+    skills = []
```

Comment:
checklist.md
2. `skills` variable should be an attribute of the instance not class.